action-templates/actions/action-checkout/action.yml

```
@@ -1,8 +1,27 @@
+    description: Defines filters applied to detected changed files.
+outputs:
+  java:
+    description: "name of the artifact upload"
+    value: ${{ steps.filter.outputs.java }}
+  javascript-typescript-vue:
+    description: "name of the artifact upload"
+    value: ${{ steps.filter.outputs.javascript-typescript-vue }}
+  python:
+    description: "name of the artifact upload"
+    value: ${{ steps.filter.outputs.python }}
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)